app/pages/lab/mobile/mobile_previewer.jsx

```diff
@@ -0,0 +1,141 @@
+                <div style={{ backgroundColor: zooTheme.colors.teal.mid }} >
+                    <div style={{display: 'flex', justifyContent: 'space-between', paddingTop: 32, paddingLeft: 25, paddingRight: 25 }} >
+                        <i className="fa fa-chevron-left" style={{margin: 'auto 0', fontSize: 26, color: 'white'}} />
+                        <span style={{flex: 1, textAlign: 'center', fontSize: 24, color: 'white' }}> {this.props.projectTitle} </span>
+                        <i className="fa fa-bars" style={{margin: 'auto 0', fontSize: 26, color: 'white'}} />
+                    </div>
+                </div>
+                {/* question view */}
+                <div style={{flexDirection: 'column'}} >
+                    <div style={{display: 'flex', justifyContent: 'center', paddingTop: 25 }}>
+                        <div style={styles.questionView} >
+                            {/* Question Header */}
+                            <div style={{display: 'flex' }}>
+                                <div style={{display: 'flex', flex: 1, justifyContent: 'center'}}>
+                                    <span style={{fontSize: 14, color: 'black', paddingTop: 15, paddingBottom: 15 }}> Question </span>
+                                </div>
+                                <div style={styles.tutorialContainer}>
+                                    <span style={{fontSize: 14, color: '#5C5C5C', paddingTop: 15, paddingBottom: 15 }}> Tutorial </span>
+                                </div>
+                            </div>
+                            {/* Question */}
+                            <div style={{ display: 'flex', paddingLeft: 15, paddingRight: 15, paddingTop: 10, paddingBottom: 15 }} >
+                                <span> {this.props.question} </span>
+                            </div>
+                            {/* image views */}
+                            <ImageComponent />
+                        </div>
+                    </div>
+                    <div style={{paddingTop: 15, paddingBottom: 15, display: 'flex', justifyContent: 'center' }}>
+                        <span style={styles.needSomeHelpStyle}> NEED SOME HELP WITH THIS TASK? </span>
+                    </div>
+                    { this.renderButtonContainer() }
+                </div>
+            </div>
+        </section>
+    );
+  }
+}
+
+const ImageComponent = () => {
+    return (
+        <div style={styles.behindImageContainer}>
+            <img src="https://i.ytimg.com/vi/AzkEf84J6aY/hqdefault.jpg" style={{ maxHeight: '210px', maxWidth: '265px', objectFit: 'contain', flex: 1 }} />
+            <img src="https://images.pexels.com/photos/356378/pexels-photo-356378.jpeg" style={styles.inFrontImage} />
+        </div>
+    )
+}
+
+const TealBox = ({text}) => {
+    return (
+        <div style={styles.tealBoxContainer}>
+            <span style={{color: 'white'}}> {text} </span>
+        </div>
+    )
+}
+
+const styles = {
```

Comment:
Hard to break habits. This works alright for now, but if merged into PFE master at some point, these styles would need to be converted to classes.

Reply:
🖕